### ISSUE TITLE
The uploaded video is different from the original video

### ISSUE THREAD

**Issue by LHY-sudo** [lensterxyz/lenster#1869]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I am going to send a short video on Lenster, but it will become another video after a few seconds of uploading. Today I have encountered this situation twice. I think this should be a bug. Please fix it as soon as possible.

### Expected Behavior

The post should have been the video I uploaded, but it became another one

### Steps To Reproduce

I uploaded the video according to the normal operation. After a few seconds of successful upload, it became another video. I can clearly see the content of this video without posting

### What platform(s) does this occur on?

Web

### What browser(s) does this occur on?

Chrome

### Anything else?

Video ready for release:
![微信截图_20230227164840](https://user-images.githubusercontent.com/63576758/221517023-cd183ed7-08b0-4d0d-bb1e-ef663c15492f.png)

Uploaded video：
![微信截图_20230227164810](https://user-images.githubusercontent.com/63576758/221517187-2c3a10fd-1684-4b1f-b326-134e9f84ded1.png)

Post-published video:
![微信截图_20230227164951](https://user-images.githubusercontent.com/63576758/221517284-d7691e02-a1ca-4671-85d5-0b17faf1bc3f.png)

**Comment by bigint** [comment]
weird issue, do you have any screencast for this 🙇🏼

**Comment by LHY-sudo** [comment]
> weird issue, do you have any screencast for this 🙇🏼

I have explained in the above picture that the first one is the video I want to upload, and the second one is the video after uploading, but we can see that it is completely different. At that time, I saw this video, and the content in it is completely different from that I uploaded. The third one is the video after I published it, and it changed back to the video I want to upload. I have met this situation twice, because I am not a computer professional, So I can't find out the specific reason. I can only tell you that the problem happened.
